components/list/list.js

```diff
@@ -57,7 +57,7 @@ class List extends LitElement {
 	}
 
 	render() {
-		const role = !this.grid ? 'list' : undefined;
+		const role = !this.grid ? 'list' : 'application';
```

Comment:
This will make the `ifDefined` directive in the template unnecessary, so it could be removed along with the associated import.  Be good to get Maya's approval on this too.